RELEASE-NOTES.md

```diff
@@ -4,6 +4,69 @@ section: overview
+- [@patternfly/react-icons: 3.15.0](https://www.npmjs.com/package/@patternfly/react-icons/v/3.15.0)
+
+### Components:
+- **Context selector:** Updated signature for context selector ([#3697](https://github.com/patternfly/patternfly-react/pull/3697))
+- **Data toolbar:** Updated html structure to match core ([#3701](https://github.com/patternfly/patternfly-react/pull/3710))
+- **Popover:** Added min-width override property ([#3601](https://github.com/patternfly/patternfly-react/pull/3601))
+- **Radio:** Added support for optional description ([#3621](https://github.com/patternfly/patternfly-react/pull/3621))
+- **Select:** Handled undefined default for checkbox select ([#3711](https://github.com/patternfly/patternfly-react/pull/3711))
+- **Simple list:** Added Simple list component ([#3645](https://github.com/patternfly/patternfly-react/pull/3645))
+- **Slider:** Added limit for lowest possible value of slider ([#3660](https://github.com/patternfly/patternfly-react/pull/3660))
+- **Wizard:** Added type to wizard context ([#3572](https://github.com/patternfly/patternfly-react/pull/3572))
+
+### Catalog view extension
+- **Catalog tile:** 
+  - Adjusted max width of header image ([#3628](https://github.com/patternfly/patternfly-react/pull/3628))
+  - Reduced padding between badge and logo ([#3644](https://github.com/patternfly/patternfly-react/pull/3644))
+  - Fixed styles that stopped working ([#3635](https://github.com/patternfly/patternfly-react/pull/3635))
+  - Fixed linting errors ([#3714](https://github.com/patternfly/patternfly-react/pull/3714))
+
+### Table:
+- Added inline edit to table ([#3058](https://github.com/patternfly/patternfly-react/issues/3058))
+- Reverted IHeaderRow interface change ([#3746](https://github.com/patternfly/patternfly-react/pull/3746))
+- Fixed that column functions are not always equal ([#3612](https://github.com/patternfly/patternfly-react/pull/3612))
+- Added support for truncated column headers ([#3729](https://github.com/patternfly/patternfly-react/pull/3729))
+
+### Other:
+- **Build:**
+  - Updated CircleCi config for pricing ([#3654](https://github.com/patternfly/patternfly-react/pull/3654))
+  - Made change to use machine instance ([#3712](https://github.com/patternfly/patternfly-react/pull/3712))
+  - Fixed typo, added md lint to CircleCi ([#3692](https://github.com/patternfly/patternfly-react/pull/3692))
+- **Chore:** 
+  - Phrased non production components as beta ([#3663](https://github.com/patternfly/patternfly-react/pull/3663))
+  - Linted React topology ([#3721](https://github.com/patternfly/patternfly-react/pull/3721))
+  - For icons made change to use absolute import paths ([#3517](https://github.com/patternfly/patternfly-react/pull/3517))
+  - Fixed broken link in issue template ([#3634](https://github.com/patternfly/patternfly-react/pull/3634))
+- **Ci:** Parallelized tests and run in PRs ([#3627](https://github.com/patternfly/patternfly-react/pull/3627))
```

Comment:
can we move this and the one below to their own bulleted line as we have been doing in other release notes?